domains/Discussions/Database/Migrations/2020_10_19_175943_create_question_answers_table.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Domains\Accounts\Models\User;
+use Domains\Discussions\Models\Question;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateQuestionAnswersTable extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('question_answers', function (Blueprint $table) {
+            $table->id();
+            $table->foreignIdFor(User::class, 'author_id');
+            $table->foreignIdFor(Question::class, 'question_id');
+            $table->text('content');
+            $table->timestampsTz();
+            $table->softDeletesTz();
+
+            $table->index(['question_id', 'author_id', 'created_at']);
```

Comment:
I was thinking about using separate indexes for each column, but since I'm no expert in database performance, I need to ask if we have better performance by using a compound index of three columns? Will it work ok?

Reply:
I use it this way because it is represented in the Laravel documentation.

![image](https://user-images.githubusercontent.com/7970196/97082036-8ad4a400-15fe-11eb-91e4-664b032cb7b3.png)

Reply:
But I can change it if necessary

Reply:
Hmm unless we want to query these 3 columns together and use an index there, I think we need to go with separate indexes.

Reply:
Sorry, seeing the laravel documentation I see that this way is to create composite indexes, which I think is not the case.

![image](https://user-images.githubusercontent.com/7970196/97082109-249c5100-15ff-11eb-965b-0cc85d44059a.png)

Reply:
Yes, I also think so. Changed!